File: scala.leap.motion/src/tutorial/scalaone/Sample.scala
package tutorial.scalaone

import java.io.IOException

import com.leapmotion.leap.Controller

object Sample {

  def main(args: Array[String]) {
    val listener: SampleListener = new SampleListener
    val controller: Controller = new Controller
    controller.addListener(listener)
    println("Press Enter to quit...")
    try {
      System.in.read
    }
    catch {
      case e: IOException => {
        e.printStackTrace
      }
    }
    controller.removeListener(listener)
  }

}

File: java.leap.motion/src/tutorial/javaone/Sample.java
package tutorial.javaone;

import com.leapmotion.leap.*;

import java.io.IOException;

public class Sample {
  public static void main(String[] args) {
    // Create a sample listener and controller
    SampleListener listener = new SampleListener();
    Controller controller = new Controller();
    controller.addListener(listener);
    // Keep this process running until Enter is pressed
    System.out.println("Press Enter to quit...");
    try {
      System.in.read();
    } catch (IOException e) {
      e.printStackTrace();
    }

    // Remove the sample listener when done
    controller.removeListener(listener);
  }
}
